src/article-list-view.js

```diff
@@ -0,0 +1,18 @@
+(function(exports) {
+  function ArticleListView(articleList) {
+    this.articleList = articleList;
+  }
+
+  ArticleListView.prototype.returnHTML = function() {
+    return this.articleList.getList().reduce((acc, val) => {
```

Comment:
Can we use the logic inside of `ArticleView` to help us render the HTML for each individual article?